xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -47,6 +47,8 @@ using namespace EPG;
 /** sleep time in milliseconds when no auto-configured add-ons were found */
 #define PVR_CLIENT_AVAHI_SLEEP_TIME_MS     (250)
 
+const int CPVRClients::CLIENT_LOAD_TIMEOUT = 5000;
```

Comment:
just #define like the line above please, and add _MS to the name please and a comment. it's only used within this class anyway